### PR TITLE
Fee recipient docs

### PR DESCRIPTION
**Pull request by james-prysm**
The initial take on fee recipient docs.

In the future, we will add the fee recipient APIs for the validator to the docs.

### CODE REVIEW

File: website/docs/prysm-usage/fee-recipient.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+id: fee-recipient
+title: Fee Recipient
+sidebar_label: fee recipient
+---
+
+# Fee Recipient
+
+Fee Recipient is the feature that allows users receive gas fees when proposing blocks. These gas rewards are some times know as the inclusion rewards.
+
+:::tip Post Bellatrix Fork ( The Merge )
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:11]:
```suggestion
:::tip Post-Merge
```
- **Assumption** (may be wrong): "The Merge" is more recognizable than "Post Bellatrix Fork".

File: website/docs/prysm-usage/fee-recipient.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+This feature will only activate Post Bellatrix Fork. Currently stakers do not receice gas fees from proposing blocks.
+:::
+
+:::warning Known Release Bug
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:16]: What does it mean to "set a default fee recipient through the validator client"? 

Does this mean that "the default fee recipient is not working"?

Or does this mean that this configuration workflow doesn't work in the current release?

Recommend splitting into two clear statements: "X doesn't work. It's fixed in Y."

File: website/docs/prysm-usage/fee-recipient.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:19]:
```suggestion
:::warning Incorrectly configured Fee Recipient settings could lead to a loss of earnings.
```
If possible, detail exactly what the worst-case is - configuration issues here won't impact anything other than these new gas fee rewards, right? Even in the worst case?

File: website/docs/prysm-usage/fee-recipient.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+Protect and review your Fee Recipient settings.
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:21]:
```suggestion
To ensure that your Fee Recipient settings are correctly configured, ___________________.
```
meta-feedback: we want to give readers clear and unambiguous instructions.

File: website/docs/prysm-usage/fee-recipient.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Fee Recipient settings will default to the burn address ( 0x000....0 ) so it is important to set it to a hex address where you can be paid this.
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:23]:
```suggestion
If you don't configure your Fee Recipient settings, the Fee Recipient address will default to the burn address (`0x000....0`). Be sure to specify a valid Ethereum wallet address (that you own) as the fee recipient. 
```
**assumption**: most users will recognize "Ethereum wallet address"; "hex address" is ambiguous.
meta-feedback: we want to use precise and unambiguous terminology when possible.

File: website/docs/prysm-usage/fee-recipient.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
+
+Protect and review your Fee Recipient settings.
+
+Fee Recipient settings will default to the burn address ( 0x000....0 ) so it is important to set it to a hex address where you can be paid this. 
+
+Please also make sure the address is yours when it is set, otherwise it will be sent to someone else's address.
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:25]:
```suggestion
```
addressed above ^^

File: website/docs/prysm-usage/fee-recipient.md (hunk 5 above)
Comment (symbolpunk) [reply website/docs/prysm-usage/fee-recipient.md:23]: Recommend moving this down to the step at which the address is specified within the config.

File: website/docs/prysm-usage/fee-recipient.md (hunk 4 above)
Comment (symbolpunk) [reply website/docs/prysm-usage/fee-recipient.md:21]: Recommend moving this to bottom of doc as a last step.

File: website/docs/prysm-usage/fee-recipient.md (hunk 6 above)
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:25]:
```suggestion
```
addressed above ^^

File: website/docs/prysm-usage/fee-recipient.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+:::
+
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:27]: Recommend including some introductory text here along the lines of "This new feature introduces a configuration file that lives _________________. Your client will read this file when ______________. 

To configure Fee Recipient, first ____________________ (create the file? add configuration boilerplate to an existing file?). Then, adjust the settings as needed. The following settings can be configured: (insert a table here for each parameter / flag value with a brief description)."

File: website/docs/prysm-usage/fee-recipient.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+( i.e. --fee-recipient-config-url=http://example.com/api/getFeeRecipient)
+
+suggested fee recipient flag will override the configuration file.
+
+you can not use both the file and url flags at the same time.
+
+the validator client will send the fee recipient information at startup for all public keys that are active
+as well as when the public key becomes active. It does this by calling the beacon api `prepareBeaconProposer`
+
+### Fee Recipient Config File based on Teku Implementation
```
Comment (symbolpunk) [inline website/docs/prysm-usage/fee-recipient.md:46]: When does a reader need to care about this section?

Comment (james-prysm) [reply website/docs/prysm-usage/fee-recipient.md:46]: if they use the fee recipient config file or url this just shows the format for that file.

File: website/docs/prysm-usage/fee-recipient.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+| `suggested-fee-recipient`   | Sets a default eth address for ALL validators.  Example: `--suggested-fee-recipient=0x046Fb65722E6b0000012BFEBf6177F1D2e9758D9`   Note: This setting overrides the two `config` options below. If you set this, the `config` options will be ignored |
+| `fee-recipient-config-file` | Sets the file location for the Fee Recipient JSON configuration.   Example: `--fee-recipient-config-file=./fee_recipient_config.json`                                                                                                                |
+| `fee-recipient-config-url`  | Sets a URL for a remote Fee Recipient JSON configuration.  Example: `--fee-recipient-config-url=http://example.com/api/getFeeRecipient`                                                                                                              |
+
+suggested fee recipient flag will override the configuration file.
+
+you can not use both the file and url flags at the same time.
+
+the validator client will send the fee recipient information at startup for all public keys that are active
```
Comment (rauljordan) [inline website/docs/prysm-usage/fee-recipient.md:47]: No need to go into implementation detail here. Let's keep it simple for the user

**Review by rauljordan — APPROVED** [review]

**Pull request merged**